index.js

```diff
@@ -47,6 +47,12 @@ ExportNodeModules.prototype.apply = function(compiler) {
         });
     });
 
+    // early out if no modules found
+    if (npmModules.size) {
+      callback();
+      return;
+    }
+
     Array.from(npmModules.keys())
```

Comment:
When this is empty, the for each and inner logic never executes. Then, on line 73 
https://github.com/ubilabs/webpack-node-modules-list/blob/1d6ecdeb1bab9aef5f6fa30c268636056c3fafd3/index.js#L73

The value of `extension` is `undefined`, producing `license.undefined`.